lib/y-richtext.coffee

```diff
@@ -215,10 +235,16 @@ class YRichText extends BaseClass
       throw new Error "This part of code must not be reached!"
 
   insertHelper : (position, content) ->
-    if content?
-      @_get("characters").insertContents position, content.split("") # convert content to an array
+    as_array =
+      if typeof content == "string"
```

Comment:
for comparisons always use `is` (equivalent to `===`). Here, it doesn't actually matter. But if you don't use the strict comparison (`===`) you may run into weird problems!